content/basics/ably.textile

```diff
@@ -0,0 +1,100 @@
+
+Ably supports both Realtime and REST integrations and provides "client library SDKs":https://ably.com/download in a number of languages to match your needs.
+
+Some of the most commonly used features are: 
+
+* "PubSub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push notifications":#push-notifications
+
+h3(#pubsub). PubSub with Channels
+
+Pub/Sub is shorthand for the Publish/Subscribe architectural pattern, a popular pattern used for realtime data delivery where the subscribers and publishers are completely decoupled. Publishers publish data, ideally to a data channel or topic, and subscribers are notified of the new data, with no need to poll servers or providers for information.
+
+Ably aggregates all its data into named units of distribution, referred to as “channels”. Channels offer a way to implement the Publish-Subscribe (Pub/Sub) architectural pattern, which is a popular pattern used for realtime data delivery. Pub/Sub message patterns let any number of publishers publish data to a channel, which could be subscribed to by any number of subscribers.
+
+Read more in our "Pub/Sub section":/channels/.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel) in real time using our realtime SDKs, or via the Ably Integrations. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Presence provides information about the activity of members entered into the presence set of a channel. Presence members announce their state to presence subscribers via presence events. For example, when entering the presence set, leaving the presence set, or updating their member data.
+
+Read more in our "Presence and occupancy section":/presence-occupancy/.
+
+h3(#message-storage-and-history). Message storage and History
+
+By default messages are only stored for two minutes on Ably’s servers, these messages can be retrieved in the event of a lost connection using the History API. There are several options available to increase this storage time. One option is to enable persistence of all messages, for typically 24 hours. These messages can also be retrieved using the History API. Persistence of the last message is also available, although this is retrieved using the rewind channel option. Finally, it is possible to persist messages to external storage services, using the various integrations that are available.
```

Comment:
This gets into the details of message storage and history by discussing channel options. I think this should be more of a high-level explanation of the feature and what it can be used for.

The other sections are referred to in terms of feature, whereas history is described as an API which stands out as being inconsistent. 

"Typically 24 hours" should be more specific too (I believe it's 24 for free tier and 72 for paid tiers). 

There's also no mention of storing messages outside of Ably (i.e. use case requires storage beyond 24/72 hours).